package.json

```diff
@@ -4,8 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "production": "npx webpack --mode=production",
-    "dev": "npx webpack --watch --mode=development"
+    "build": "npx webpack --mode=production",
+    "dev": "npx webpack --watch --mode=development",
```

Comment:
I am curious about why you are using `npx` here?

Reply:
Nothing much, I want to look into changing to npx after I do some investigative work.